BookPlayer/Player/SleepTimer.swift

```diff
@@ -39,6 +40,10 @@ final class SleepTimer {
     1800.0,
     3600.0
   ]
+  
+  public var lastInteral: TimeInterval = 300
+  
+  public var sticky = false
```

Comment:
We'll need a setter for updating the stored value when the user taps on the alert sheet, and a getter to check on the stored value when resuming playback